### PR TITLE
examples: add awslambdaechobot

### PR DESCRIPTION
**Pull request by MetalBlueberry**
Hello! This is an awslambdaechobot exmaple that just works. 

I want to put a bit more time into this later so I can explain how every piece in this example works, but for now I would like to know your opinion.

### CODE REVIEW

File: examples/awslambdaechobot/main.go [hunk 1]
```diff
@@ -0,0 +1,101 @@
+package main
+
+import (
+	"context"
+	"encoding/json"
+	"log"
+	"os"
+
+	"github.com/aws/aws-lambda-go/lambda"
+	"gopkg.in/telebot.v3"
```
Comment (suntong) [inline examples/awslambdaechobot/main.go:10]: This line seems to be redundant and can be removed.

**Comment by suntong** [comment]
Looks good now. Thanks.

@tucnak, Looks good to you as well?

File: examples/awslambdaechobot/main.go [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
```
Comment (demget) [inline examples/awslambdaechobot/main.go:10]: Please add a `tele` alias.

File: examples/awslambdaechobot/destroy.sh [hunk 3]
```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+set -e 
+set -o pipefail
+
+
+terraform -chdir=./tf destroy -input=false -var=bot_token=$TOKEN 
+
+curl -H "Content-Type: application/json"  -X POST "https://api.telegram.org/bot$TOKEN/setWebhook" -d "{
```
Comment (demget) [inline examples/awslambdaechobot/destroy.sh:8]: Make a one-line:
`curl -H "Content-Type: application/json" -X POST "https://api.telegram.org/bot$TOKEN/setWebhook" -d "{\"url\": \"\" }"`

File: examples/awslambdaechobot/main.go [hunk 4]
```diff
@@ -0,0 +1,100 @@
...
+		Token:       botToken,
+		Synchronous: true,
+	}
+
+	b, err := telebot.NewBot(pref)
+	if err != nil {
+		log.Fatal(err)
+		return
+	}
+	b.Handle(telebot.OnText, func(ctx telebot.Context) error {
```
Comment (demget) [inline examples/awslambdaechobot/main.go:28]: `c tele.Context`

**Review by demget — CHANGES_REQUESTED** [review]

**Pull request closed without merging**